XML/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ POCO_HEADERS_AUTO( SRCS ${HDRS_G})
 # If POCO_UNBUNDLED is enabled we try to find the required packages
 # The configuration will fail if the packages are not found
 if (POCO_UNBUNDLED)
+    hunter_add_package(Expat)
```

Comment:
`EXPAT_LIBRARIES` and `EXPAT_INCLUDE_DIRS` is not a relocatable configuration. You should use `EXPAT::expat` imported library:
* https://github.com/ruslo/hunter/blob/c36af908c28948c13c8f3321ee81dac53d8b8b14/examples/Expat/CMakeLists.txt#L13-L18

Documentation `pkg.Expat` is not up-to-date, feel free to send pull request with update:
* https://docs.hunter.sh/en/latest/packages/pkg/Expat.html

Just for the reference here is the relocation fix commit:
* https://github.com/ruslo/hunter/commit/8144af600cc1e1e696286500f7f57df0352dea12